var/spack/repos/spack_repo/builtin/packages/root/package.py

```diff
@@ -401,11 +430,11 @@ class Root(CMakePackage):
     depends_on("graphviz", when="+graphviz")
     depends_on("gsl", when="+gsl")
     depends_on("jemalloc", when="+jemalloc")
-    depends_on("mysql-client", when="+mysql")
+    depends_on("mysql-client", when="+mysql @:6.36")
     depends_on("openssl", when="+ssl")
     depends_on("openssl", when="+davix")  # Also with davix
     depends_on("oracle-instant-client@19.10.0.0.0", when="+oracle @:6.24.01")
-    depends_on("postgresql", when="+postgres")
+    depends_on("postgresql", when="+postgres @:6.36")
```

Comment:
When is implicit in variant.
```suggestion
    depends_on("postgresql", when="+postgres")
```